charts/datadog/values.yaml

```diff
@@ -520,6 +520,28 @@ datadog:
       # datadog.asm.iast.enabled -- Enable Application Security Management Interactive Application Security Testing by injecting `DD_IAST_ENABLED=true` environment variable to all pods in the cluster
       enabled: false
 
+  ## Continuous Profiler configuration
+  ##
+  ## Continuous Profiler is disabled by default and can be enabled by setting the `enabled` field to
+  ## either `auto` or `true` value under the `datadog.profiling` section.
+  ## Manually adding the `DD_PROFILING_ENABLED` variable to a pod will take precedence over the
+  ## value in the Helm chart.
+  ## These will only have an effect on containers that have Datadog client libraries installed,
+  ## either manually or via Single Step Instrumentation (under the `datadog.apm.instrumentation`
+  ## section).
```

Comment:
If you follow the implementation ASM did in the cluster-agent then it will only trigger on pod with SSI and not those with manually installed library. Injecting vars into manually installed pods should be done by the end of Q3 following the work done by APM on service discovery.

Reply:
I pretty much took over the comment from the ASM part as well – it, too, says "either manually or via Single Step Instrumentation" so I guess it should say the same in both places?